static/js/components/ClassificationList.jsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { Button } from "@material-ui/core";
+import Tooltip from "@material-ui/core/Tooltip";
+import GroupIcon from "@material-ui/icons/Group";
+import ListItem from '@material-ui/core/ListItem';
+import { FixedSizeList } from 'react-window';
+
+import dayjs from "dayjs";
+import relativeTime from "dayjs/plugin/relativeTime";
+
+import * as sourceActions from "../ducks/source";
+import styles from "./ClassificationList.css";
+
+dayjs.extend(relativeTime);
+
+const ClassificationList = () => {
+  const [hoverID, setHoverID] = useState(null);
+
+  const handleMouseHover = (id, userProfile, author) => {
+    if (userProfile.roles.includes("Super admin") || userProfile.username === author) {
```

Comment:
Should group admins also be able to delete?

Reply:
Good idea.